charts/hedgedoc/values.yaml

```
@@ -0,0 +1,269 @@
+# Default values for hedgedoc-apps.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  # -- Override the repository
+  repository: quay.io/hedgedoc/hedgedoc
+  # -- Override the pullPolicy
+  pullPolicy: IfNotPresent
+  # -- Override the imaae tag
+  tag: ""
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # -- Specifies whether a service account should be created
+  create: true
+  # -- Annotations to add to the service account
+  annotations: {}
+  # -- The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+podAnnotations: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+# -- Ingress configuration
+ingress:
+  enabled: false
+  className: ""
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  hosts:
+    - host: chart-example.local
+      paths:
+        - path: /
+          pathType: ImplementationSpecific
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
+# -- Service configuration
+service:
+  port: 3000
+  type: ClusterIP
+
+# -- Main configuration for hedgedoc
+hedgedoc:
+  # -- Persistent Storage
+  # Enable this if you don't want to loose media
+  persistentStorage:
+    enabled: true
+    accessModes:
+      - ReadWriteOnce
+    volumeMode: Filesystem
+    size: "10G"
+    storageClass: "default"
+  # -- Port for the pod
+  port: 3000
+  ## Environment variables for Hedgedoc
+  ## See the official docs for all variables
+  ## https://docs.hedgedoc.org/configuration/
+  env:
```

something like this would get rid of the long list of envs in the README
```suggestion
  # -- Environment variables for Hedgedoc
  # @default -- See the [official docs for all variables](https://docs.hedgedoc.org/configuration/)
  env:
```